prboom2/src/e6y.c

```diff
@@ -220,7 +218,7 @@ void e6y_assert(const char *format, ...)
   va_list argptr;
   va_start(argptr,format);
   //if (!f)
-    f = fopen("d:\\a.txt", "ab+");
+    f = M_fopen("d:\\a.txt", "ab+");
```

Comment:
Irrelevant changes from master ended up in your branch. I guess you staged them by accident?

Reply:
I saw that. The branch is more than a year old. The local file I edited and uploaded is synced to the latest master. I suppose the only way to avoid these false diffs showing up would be to delete my fork and do it all over again.

Reply:
You can revert the last commit, then run `git merge master`, commit the latest changes again, and then push. You will get a branch synced with master. I'm not good with git, maybe there is an easier way.